### PR TITLE
Tabs

### PR DESCRIPTION
**Pull request by jaruesink**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@date-io/luxon": "^2.10.6",
     "@material-ui/core": "^4.11.0",
     "@material-ui/lab": "^4.0.0-alpha.56",
-    "@material-ui/pickers": "4.0.0-alpha.12",
+    "@material-ui/pickers": "^3.2.10",
```
Comment (jaruesink) [inline package.json:29]: we need the alpha for my date-picker, can we revert this?

File: src/lib/tabs/Tabs.tsx [hunk 2]
```diff
@@ -27,8 +27,31 @@ function a11yProps(index: any) {
...
+  appBar: {
+    backgroundColor: 'rgba(0,0,0,0)',
+    boxShadow: 'none'
+  },
   root: {
-    flexGrow: 1
+    borderRadius: '4px',
+    boxShadow: '0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)',
+    flexGrow: 1,
+    fontFamily: 'lc-gilroy'
```
Comment (jaruesink) [inline src/lib/tabs/Tabs.tsx:38]: this should probably inherit also

File: src/lib/tabs/Tabs.tsx [hunk 3]
```diff
@@ -27,8 +27,31 @@ function a11yProps(index: any) {
...
 
 const useStyles = makeStyles((theme: Theme) => ({
+  appBar: {
+    backgroundColor: 'rgba(0,0,0,0)',
+    boxShadow: 'none'
+  },
   root: {
-    flexGrow: 1
+    borderRadius: '4px',
+    boxShadow: '0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)',
```
Comment (jaruesink) [inline src/lib/tabs/Tabs.tsx:36]: I like the way the card and shadows look, but it's probably too opinionated for the component. Ideally we will be able to theme this component for Nybll's menu which doesn't have a border or any box-shadows. So we could probably just have this as a flat component and whoever is implementing the component can put it into an elevated card if they want to.

File: package.json (hunk 1 above)
Comment (jaruesink) [reply package.json:29]: once you do this, you'll need to rerun yarn and resolve the yarn.lock conflict

**Comment by jaruesink** [comment]
@Carlo-Piantini can we get the underline of the tab to be set to `var(--primary-default-color, #3182ce)` instead of the Mui default pink?

File: package.json (hunk 1 above)
Comment (Carlo-Piantini) [reply package.json:29]: This is resolved!

File: src/lib/tabs/Tabs.tsx (hunk 2 above)
Comment (Carlo-Piantini) [reply src/lib/tabs/Tabs.tsx:38]: This is resolved!

File: src/lib/tabs/Tabs.tsx (hunk 3 above)
Comment (Carlo-Piantini) [reply src/lib/tabs/Tabs.tsx:36]: This is resolved!

**Comment by Carlo-Piantini** [comment]
Hey @jaruesink, these changes have all been made! Take another look and let me know if this needs anything else. Thanks!

**Pull request merged**